pkgs/development/libraries/libgourou/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchurl, pugixml, updfparser, curl, openssl, libzip, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "libgourou";
+  version = "0.8.1";
+
+  src = fetchurl {
```

Comment:
You can use `fetchzip` here.

Reply:
I have tried that before and unfortunately that doesn't result in automatic unpacking either.

> trying https://indefero.soutade.fr/p/libgourou/source/download/v0.8.1/
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100 80611    0 80611    0     0  29750      0 --:--:--  0:00:02 --:--:-- 29756
> unpacking source archive /build/v0.8.1
> do not know how to unpack source archive /build/v0.8.1

Reply:
`fetchgit` also does not work, because apparently soutade.fr requires ssh for cloning

Reply:
Try `extension = "zip"`. Also, the download resulted in a different hash for me. I have `sha256-ZKnA+Ip96G1Iy6EDqu6BIz8O90MfqOUbla0z2RNSbNI=`